website/docs/reference/glossary_style.md

```diff
@@ -0,0 +1,82 @@
+# Glossary style
```

Comment:
As I'm reading, I think this might be better as an explanation page. The descriptions are really helpful (I didn't even know what a gloss was lol) - it'd be good to go all in and have similar descriptions for all the terms.

We might eventually want to have a reference page, but this is key info and it's good to have these terms actually explained.

Reply:
If you agree that we can make this an explanation, I think some of the sections can be re-ordered to group them by similarity. We might (for example) want to start with what a glossary is (and isn't) for, then the definitions of the important terminology, then what makes a good glossary entry, and then finally the examples.

Reply:
What we could do then is define an example glossary on this page, where we can put the examples themselves. That way, they'll actually be displayed as if they are a glossary (i.e. the glossary doesn't have to be on a separate page) and then that makes some of the definitions easier to illustrate (sort of how  you did in the original txt file). What do you think?

Reply:
Sounds good to me! I look forward to seeing what you come up with. I'll keep an eye out but feel free to use the "request re-review" button (or whatever it's called) when you're ready and I'll take another look :)